Clarify go install method (#633)
timescale/docs: timescaledb/how-to-guides/backup-and-restore/timescaledb-backup.md

    *   `pg_dump`
    *   `pg_dumpall`
    *   `pg_restore`

> 1.  Download `timescaledb-backup`:

Reviewer: ```suggestion
1.  Download `timescaledb-backup`. If you see an error about 
     not being able to load packages, you can safely ignore them:
```

    ```bash
    $ go get -d github.com/timescale/timescaledb-backup/

>     # Ignore any can't load packages errors, if any.

Reviewer: ```suggestion
```

Author: Thanks for the suggestions. I'm not sure if I'm including them correctly. Can you confirm that the above two suggestions have been accepted by me?

Reviewer: If you go to the `Files Changed` tab in the GH web interface, you can click `Add suggestion to batch` for each suggestion that you want to include. Then you can click `Commit suggestions` at the top of the page when. you're ready. That creates a new commit that you can see in the `Commits` tab.

    ```
1. Build the binaries:
   ```bash
   $ cd ~/go/src/github.com/timescale/timescaledb-backup/
   $ go install ./cmd/...
   ```

>    The binaries will be available on your GOPATH.
>     Find it with:

Reviewer: ```suggestion
1.  The binaries are available on your `GOPATH`. You can find them with this command:
```

    ```bash
    $ go env
    ```

> 1. (Optional) Make the binaries available on your path:

Reviewer: ```suggestion
1. ***Optional:*** Make the binaries available on your path:
```

>      * Debian/Ubuntu:

Reviewer: Can we add instructions for other OSs, and include them in a `<code>` block?

Author: Might take me a week but I can add RHEL/CentOS (7) to this list. Windows and MacOS are beyond me.
